app/models/manageiq/providers/infra_manager/metrics_capture.rb

```diff
@@ -78,7 +78,7 @@ def capture_host_targets(ems)
   # @return [Array<Storage>] supported storages
   # hosts preloaded storages and tags
   def capture_storage_targets(hosts)
-    hosts.flat_map(&:storages).uniq.select { |s| Storage.supports?(s.store_type) & s.perf_capture_enabled? }
```

Comment:
so while we introduced `storage_type_supported_for_ssa?`, I rolled it into `supports?(:smartstate_analysis)` and basically removed all the calls.

Here, I felt `perf_capture_enabled?` wasn't part of `supports`, so I kept this separate.